packages/react-filerobot-image-editor/src/components/ToolsBar/index.jsx

```
@@ -4,8 +4,8 @@ import PropTypes from 'prop-types';
 
 /** Internal Depepdencneis */
 import { SELECT_TOOL } from 'actions';
-import { TABS_TOOLS, TOOLS_ITEMS } from 'components/tools/tools.constants';
-import { TABS_IDS } from 'utils/constants';
+import { TABS_TOOLS, TOOLS_ITEMS, AVAILABLE_ANNOTATIONS_TOOLS } from 'components/tools/tools.constants';
```

ESLint

[error] Replace `·TABS_TOOLS,·TOOLS_ITEMS,·AVAILABLE_ANNOTATIONS_TOOLS·` with `⏎··TABS_TOOLS,⏎··TOOLS_ITEMS,⏎··AVAILABLE_ANNOTATIONS_TOOLS,⏎`

(prettier/prettier)